server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheAdd.java

```diff
@@ -22,36 +22,31 @@
 
 package org.jboss.as.clustering.infinispan.subsystem;
 
-import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.OP_ADDR;
```

Comment:
I had the same issue in previous file. There were probably changes in ide-settings over time.